README.md

```diff
@@ -6,6 +6,7 @@
 PS2Link (C) 2003 Tord Lindstrom (pukko@home.se)
         (C) 2003,2004 adresd (adresd_ps2dev@yahoo.com)
         (C) 2003,2004,2005 Khaled (khaled@w-arts.com)
+        (C) 2019,2020,2021 fjtruhy (fjtrujy@gmail.com)
```

Comment:
Not very helpful, but you mistyped your name here

Reply:
:-)